### ISSUE TITLE
演示工作流报错，无法运行

### ISSUE THREAD

**Issue by xiye9325** [MinusZoneAI/ComfyUI-CogVideoX-MZ#24]
MZ大佬，我刚安装了你在Cogvideo上的模型加载节点，KJ大佬的Cogvideo也更新到了最新版本，但用你在Github示例的工作流直接报错了，仔细一对比为啥我的Cogvideo采样器跟你示例的工作流里都不一样，求解惑 ，也删掉重装过还是老版本的节点，折腾一天了毫无头绪

![微信图片_20241009202005](https://github.com/user-attachments/assets/cf40ad62-31a2-4e7d-966c-58fe1b3be409)

错误报告
# ComfyUI Error Report
## Error Details
- **Node Type:** MZ_CogVideoXLoader
- **Exception Type:** RuntimeError
- **Exception Message:** Error(s) in loading state_dict for AutoencoderKLCogVideoX:
	Missing key(s) in state_dict: "encoder.conv_in.conv.weight", "encoder.conv_in.conv.bias", "encoder.down_blocks.0.resnets.0.norm1.weight", "encoder.down_blocks.0.resnets.0.norm1.bias", "encoder.down_blocks.0.resnets.0.norm2.weight", "encoder.down_blocks.0.resnets.0.norm2.bias", "encoder.down_blocks.0.resnets.0.conv1.conv.weight", "encoder.down_blocks.0.resnets.0.conv1.conv.bias", "encoder.down_blocks.0.resnets.0.conv2.conv.weight", "encoder.down_blocks.0.resnets.0.conv2.conv.bias", "encoder.down_blocks.0.resnets.1.norm1.weight", "encoder.down_blocks.0.resnets.1.norm1.bias", "encoder.down_blocks.0.resnets.1.norm2.weight", "encoder.down_blocks.0.resnets.1.norm2.bias", "encoder.down_blocks.0.resnets.1.conv1.conv.weight", "encoder.down_blocks.0.resnets.1.conv1.conv.bias", "encoder.down_blocks.0.resnets.1.conv2.conv.weight", "encoder.down_blocks.0.resnets.1.conv2.conv.bias", "encoder.down_blocks.0.resnets.2.norm1.weight", "encoder.down_blocks.0.resnets.2.norm1.bias", "encoder.down_blocks.0.resnets.2.norm2.weight", "encoder.down_blocks.0.resnets.2.norm2.bias", "encoder.down_blocks.0.resnets.2.conv1.conv.weight", "encoder.down_blocks.0.resnets.2.conv1.conv.bias", "encoder.down_blocks.0.resnets.2.conv2.conv.weight", "encoder.down_blocks.0.resnets.2.conv2.conv.bias", "encoder.down_blocks.0.downsamplers.0.conv.weight", "encoder.down_blocks.0.downsamplers.0.conv.bias", "encoder.down_blocks.1.resnets.0.norm1.weight", "encoder.down_blocks.1.resnets.0.norm1.bias", "encoder.down_blocks.1.resnets.0.norm2.weight", "encoder.down_blocks.1.resnets.0.norm2.bias", "encoder.down_blocks.1.resnets.0.conv1.conv.weight", "encoder.down_blocks.1.resnets.0.conv1.conv.bias", "encoder.down_blocks.1.resnets.0.conv2.conv.weight", "encoder.down_blocks.1.resnets.0.conv2.conv.bias", "encoder.down_blocks.1.resnets.0.conv_shortcut.weight", "encoder.down_blocks.1.resnets.0.conv_shortcut.bias", "encoder.down_blocks.1.resnets.1.norm1.weight", "encoder.down_blocks.1.resnets.1.norm1.bias", "encoder.down_blocks.1.resnets.1.norm2.weight", "encoder.down_blocks.1.resnets.1.norm2.bias", "encoder.down_blocks.1.resnets.1.conv1.conv.weight", "encoder.down_blocks.1.resnets.1.conv1.conv.bias", "encoder.down_blocks.1.resnets.1.conv2.conv.weight", "encoder.down_blocks.1.resnets.1.conv2.conv.bias", "encoder.down_blocks.1.resnets.2.norm1.weight", "encoder.down_blocks.1.resnets.2.norm1.bias", "encoder.down_blocks.1.resnets.2.norm2.weight", "encoder.down_blocks.1.resnets.2.norm2.bias", "encoder.down_blocks.1.resnets.2.conv1.conv.weight", "encoder.down_blocks.1.resnets.2.conv1.conv.bias", "encoder.down_blocks.1.resnets.2.conv2.conv.weight", "encoder.down_blocks.1.resnets.2.conv2.conv.bias", "encoder.down_blocks.1.downsamplers.0.conv.weight", "encoder.down_blocks.1.downsamplers.0.conv.bias", "encoder.down_blocks.2.resnets.0.norm1.weight", "encoder.down_blocks.2.resnets.0.norm1.bias", "encoder.down_blocks.2.resnets.0.norm2.weight", "encoder.down_blocks.2.resnets.0.norm2.bias", "encoder.down_blocks.2.resnets.0.conv1.conv.weight", "encoder.down_blocks.2.resnets.0.conv1.conv.bias", "encoder.down_blocks.2.resnets.0.conv2.conv.weight", "encoder.down_blocks.2.resnets.0.conv2.conv.bias", "encoder.down_blocks.2.resnets.1.norm1.weight", "encoder.down_blocks.2.resnets.1.norm1.bias", "encoder.down_blocks.2.resnets.1.norm2.weight", "encoder.down_blocks.2.resnets.1.norm2.bias", "encoder.down_blocks.2.resnets.1.conv1.conv.weight", "encoder.down_blocks.2.resnets.1.conv1.conv.bias", "encoder.down_blocks.2.resnets.1.conv2.conv.weight", "encoder.down_blocks.2.resnets.1.conv2.conv.bias", "encoder.down_blocks.2.resnets.2.norm1.weight", "encoder.down_blocks.2.resnets.2.norm1.bias", "encoder.down_blocks.2.resnets.2.norm2.weight", "encoder.down_blocks.2.resnets.2.norm2.bias", "encoder.down_blocks.2.resnets.2.conv1.conv.weight", "encoder.down_blocks.2.resnets.2.conv1.conv.bias", "encoder.down_blocks.2.resnets.2.conv2.conv.weight", "encoder.down_blocks.2.resnets.2.conv2.conv.bias", "encoder.down_blocks.2.downsamplers.0.conv.weight", "encoder.down_blocks.2.downsamplers.0.conv.bias", "encoder.down_blocks.3.resnets.0.norm1.weight", "encoder.down_blocks.3.resnets.0.norm1.bias", "encoder.down_blocks.3.resnets.0.norm2.weight", "encoder.down_blocks.3.resnets.0.norm2.bias", "encoder.down_blocks.3.resnets.0.conv1.conv.weight", "encoder.down_blocks.3.resnets.0.conv1.conv.bias", "encoder.down_blocks.3.resnets.0.conv2.conv.weight", "encoder.down_blocks.3.resnets.0.conv2.conv.bias", "encoder.down_blocks.3.resnets.0.conv_shortcut.weight", "encoder.down_blocks.3.resnets.0.conv_shortcut.bias", "encoder.down_blocks.3.resnets.1.norm1.weight", "encoder.down_blocks.3.resnets.1.norm1.bias", "encoder.down_blocks.3.resnets.1.norm2.weight", "encoder.down_blocks.3.resnets.1.norm2.bias", "encoder.down_blocks.3.resnets.1.conv1.conv.weight", "encoder.down_blocks.3.resnets.1.conv1.conv.bias", "encoder.down_blocks.3.resnets.1.conv2.conv.weight", "encoder.down_blocks.3.resnets.1.conv2.conv.bias", "encoder.down_blocks.3.resnets.2.norm1.weight",

**Comment by wailovet** [comment]
VAE模型对了吗 看起来好像是vae模型加载失败了

**Comment by xiye9325** [comment]
@wailovet 好像还真是这个问题，谢谢解答，但这个好像只是问题之一，调整回VAE模型后采样器还是红框报错，而且我这里更新的KJ大佬的节点采样器好像跟MZ大佬演示的采样器节点不太一样

**Comment by xiye9325** [comment]
这是在运行窗口弹出的错误报告

* CogVideoSampler 57:
  - Value not in list: scheduler: 'DPM' not in ['DPM++', 'Euler', 'Euler A', 'PNDM', 'DDIM', 'CogVideoXDDIM', 'CogVideoXDPMScheduler', 'SASolverScheduler', 'UniPCMultistepScheduler', 'HeunDiscreteScheduler', 'DEISMultistepScheduler', 'LCMScheduler']
Output will be ignored
Failed to validate prompt for output 44:
Output will be ignored

**Comment by wailovet** [comment]
重新点选一下采样器那边的参数

**Comment by xiye9325** [comment]
非常感谢你的耐心解答，点选后又出现了新的错误，真是困难重重啊，我已经快到放弃的边缘了，花了一天时间从入门到入土😅  

# ComfyUI Error Report
## Error Details
- **Node Type:** CogVideoSampler
- **Exception Type:** RuntimeError
- **Exception Message:** mat1 shape (452x4096 must be divisible by 16
## Stack Trace
```
  File "G:\ComfyUI-aki\ComfyUI-aki-v1.3\execution.py", line 317, in execute
    output_data, output_ui, has_subgraph = get_output_data(obj, input_data_all, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)

  File "G:\ComfyUI-aki\ComfyUI-aki-v1.3\execution.py", line 192, in get_output_data
    return_values = _map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)

  File "G:\ComfyUI-aki\ComfyUI-aki-v1.3\execution.py", line 169, in _map_node_over_list
    process_inputs(input_dict, i)

  File "G:\ComfyUI-aki\ComfyUI-aki-v1.3\execution.py", line 158, in process_inputs
    results.append(getattr(obj, func)(**inputs))

  File "G:\ComfyUI-aki\ComfyUI-aki-v1.3\custom_nodes\ComfyUI-CogVideoXWrapper\nodes.py", line 953, in process
    latents = pipeline["pipe"](

  File "G:\ComfyUI-aki\ComfyUI-aki-v1.3\python\lib\site-packages\torch\utils\_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)

**Comment by wailovet** [comment]
宽高和长度都选对了吗,给个图看看

**Comment by xiye9325** [comment]
经过我一下午的瞎折腾好像已经跑通了，虽然我还不知道具体问题到底出在什么地方，总之感谢耐心解答，谢谢
